packages/temporal-time-utils/README.md

```diff
@@ -145,3 +149,31 @@ await handle.signal(stateSignal, "RUNNING" as ScheduleWorkflowState); // resume
 await handle.cancel(); // stop schedule workflow completely
 await handle.query(stateQuery); // get wf state (running, paused, or stopped)
 ```
+
+## `Entity`
+
+This special class packages an indefinitely long lived Workflow and the Signal and Query that go with updating it. It correctly handles the pending Signals and `continueAsNew`, and calls `continueAsNew` at least once a day as recommended by Temporal.
+
+```ts
+import { Entity } from "temporal-time-utils";
+
+interface Input { /* define your workflow input type here */ }
+interface Update { /* define your workflow update type here */ }
+const entity = new Entity<Input, Update>({
+  activity: 'MyActivityName'
+  activityOptions: {
+    startToCloseTimeout: '1 minute',
+  }
+})
```

Comment:
This would be a workflow code

Reply:
yeah good point

Reply:
actually.. if i dont make it workflow code, then i dont have to register it separately.